.github/workflows/pull-request-review.yml

```
@@ -0,0 +1,46 @@
+            }
+
+            // Sort reviews by submission time in descending order
+            codeRabbitReviews.sort((a, b) => new Date(b.submitted_at) - new Date(a.submitted_at));
+            const latestReview = codeRabbitReviews[0];
+
+            // Check if the latest CodeRabbit review is "APPROVED"
+            if (latestReview.state !== 'APPROVED') {
+              core.setFailed('ERROR: CodeRabbit approval is required before merging this PR.');
+            } else {
+              console.log('Success: CodeRabbit has approved this PR for Talawa-API.');
+            }
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)